docs/content/preview/faq/yugabyte-platform.md

```diff
@@ -123,3 +138,88 @@ Yes, you can control what YugabyteDB Anywhere is spinning up. For example:
+1. Run the installation again, or the command `node-agent node configure`. Running the command is faster as it does not install node agent again.
+1. Start the `systemd` service.
+
+For YBA version 2.18.5 and later, do the following:
+
+1. Remove the node instance from the provider if it is already added using the command `node-agent node delete-instance`.
+1. Run the command `node-agent node configure`.
+
+As long as the IP does not change, the node agent does not try to register again.
+
+Note that the first step to change the provider is very important because after the provider configuration information changes in the config file as part of the second step, the node agent can't find the node to delete it as the scope is always with the provider and availability zone.
+
+### How does a node agent perform preflight checks?
+
+A node agent utility (node-agent) does the following when the preflight-check command is run:
+
+1. Runs the [preflight_check](https://github.com/yugabyte/yugabyte-db/blob/master/managed/node-agent/resources/preflight_check.sh) script.
+1. Collects the output and converts it to a well-formatted JSON payload.
+1. Sends the payload to YBA for validation. YBA has preflight check threshold values defined in the [runtime configuration](../../yugabyte-platform/administer-yugabyte-platform/manage-runtime-config/) of the provider. The prefix path for the config key name is `yb.node_agent.preflight_checks`. The values can be changed if needed.
+
+### How to disable a node agent?
+
+A node agent can be disabled anytime at the [provider scope](../../yugabyte-platform/administer-yugabyte-platform/manage-runtime-config/). It defaults to SSH for remote communication. The config key name is `yb.node_agent.client.enabled`.
+
+### How to change the node agent default port of 9070?
+
+Use the global [runtime configuration](../../yugabyte-platform/administer-yugabyte-platform/manage-runtime-config/) key `yb.node_agent.server.port`. The change is reflected only on newly created node agents.
+
+### Is it okay to manually edit the configuration file for node agent?
+
+It is not recommended to do so because editing the file can interfere with the self-upgrade workflow. If it is a minor change, it is better to stop the node agent service first to keep YBA away from starting the upgrade process.
+
+### How does YBA determine that a node instance record maps to a node agent entry if they are not related?
+
+YBA uses the IP address to identify a node instance. The IP can be a DNS from YBA version 2.18.5 and later.
+
+For YBA versions later than 2.18.5, a bind address that defaults to the IP will be added in case a DNS is supplied and the node agent has to listen to a specific interface IP.
+
+### How does YBA clean up node agents for on-prem non-manual nodes?
+
+YBA removes the node agent entry when the node is cleaned up. You can choose to leave the node agent service running, but when the node is again re-used, the node agent is re-installed.
```

Comment:
when the node is cleaned up => when YBA releases the node back to the node instances pool of the provider.